tests/lib/snaps/network-observe-consumer/bin/consumer

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+
+import subprocess
+import sys
+
+def run():
+    print(subprocess.check_output("netstat -lnt", shell=True))
```

Comment:
You could use  `subprocess.check_call(["netstat", "-lnt"])` here (which is slightly shoter). The check_call() will simply use the existing stdout, stderr. But this is fine of course, just a few bytes longer :)